### PR TITLE
Fix class names holding diff. flavors of partition information

### PR DESCRIPTION
**Pull request by SreeramGarlapati**
Rename EventHubsPartitionRuntimeInformation to PartitionRuntimeInformation
Rename ReceiverRuntimeInformation to PartitionEndOfStreamInformation
Rename getLastSequenceNumber to getLast**Endqueued**SequeunceNumber

### CODE REVIEW

File: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/PartitionEndOfStreamInformation.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 import java.time.Instant;
 
-public final class ReceiverRuntimeInformation {
+public final class PartitionEndOfStreamInformation {
```
Comment (sabeegrewal) [inline azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/PartitionEndOfStreamInformation.java]: How about `EndOfPartitionInformation`?

File: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/PartitionEndOfStreamInformation.java [hunk 2]
```diff
@@ -25,7 +25,7 @@ public String getPartitionId() {
         return this.partitionId;
     }
 
-    public long getLastSequenceNumber() {
+    public long getLastEnqueuedSequenceNumber() {
```
Comment (sabeegrewal) [inline azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/PartitionEndOfStreamInformation.java]: I think the fact that the event has been enqueued is obvious. `lastSequenceNumber` seems clear to me, and the extra bit is unneeded verbosity. 

At the end of the day, I'm fine with either, but my preference would be that all other clients get updated to `getLastSequenceNumber`.

File: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/PartitionEndOfStreamInformation.java (hunk 1 above)
Comment (sabeegrewal) [inline azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/PartitionEndOfStreamInformation.java]: How about `EndOfPartitionInformation`?

File: azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/PartitionEndOfStreamInformation.java (hunk 2 above)
Comment (SreeramGarlapati) [reply azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/PartitionEndOfStreamInformation.java]: on a stream first and last are relative - last depends on whats your first - `lastEnqueued` is an attempt to make it explicit... 

all other clients also have getLastSequenceNumber only - just that other properties (like offset, and timestamp) in this class has `Enqueued` in them - it felt a little different for this property to not have it!

**Pull request merged**